src/helpers/getPlaystreakQuestStatus.ts

```diff
@@ -23,6 +23,8 @@ export function getPlaystreakQuestStatus(
     (gameIsInfinitelyCompleteable || gameHasMoreFiniteCompletionsPossible)
   ) {
     return 'READY_FOR_CLAIM'
+  } else if (quest.status === 'COMPLETED') {
+    return undefined
```

Comment:
Why undefined instead of addind another option to the type? I find a value like `NOT_QUALIFIED` better than the absence of value.

Reply:
I didn't want it to be confused with a `QuestLogInfo['state']` value. `undefined` in this case means the quest shouldn't be shown to the user. 

I do see where your coming from and would make the change if I weren't under a bit of a time crunch. This is a bit of a nitpick though